compiler/src/dmd/s2ir.d

```diff
@@ -164,16 +164,10 @@ private extern (C++) class S2irVisitor : Visitor
 
         if (s.elsebody)
         {
-            const isFalse = s.condition.toBool().hasValue(true);
-            irs.falseBlock += isFalse;
```

Comment:
Are you abandoning this folly? So this is now a dead field?  It's unclear how this back-end and the front-end change relate to the issue.